implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -363,7 +363,9 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
             contextInfo.setPublicKeyContent(mpJwtPublicKey.get());
         } else if (mpJwtLocation.isPresent() && !NONE.equals(mpJwtLocation.get())) {
             contextInfo.setPublicKeyLocation(mpJwtLocation.get().trim());
-        } else if (mpJwtDecryptKeyLocation.isPresent() && !NONE.equals(mpJwtDecryptKeyLocation.get())) {
```

Comment:
Was it a rebase issue ? Probably, thanks for fixing it

Reply:
Not sure what it was, but it was failing one of the tests :)